py_find_injection/__init__.py

```diff
@@ -122,6 +122,7 @@ def main():
         epilog='Exit status is 0 if all files are okay, 1 if any files have an error. Errors are printed to stdout'
     )
     parser.add_argument('--version', action='version', version='%(prog)s ' + __version__)
+    parser.add_argument('--ci', action='store', required=False)
```

Comment:
we can't use `--ci` if we're sending this upstream, this is kind of a datadog related change. The original author may not care about it.